src/dataDisplay/Number/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+const DEFAULT_SHOW_DECIMALS = true;
+const DEFAULT_SHOW_THOUSAND_SEPARATORS = false;
+
+type FormatNumberOptions = {
+  roundingType?: 'up' | 'down';
+  showDecimals?: boolean;
+  decimalPlaces?: number;
+  showThousandSeparators?: boolean;
+  currency?: string;
+  as?: 'p' | 'span';
```

Comment:
Should we include HTML formatting elements?

Reply:
After thinking about it, they don't make sense to have.

Reply:
Why?

Reply:
The component would theoretically be used within an element. What do you think?

Reply:
I'd prefer to do `<strong>0.1 ETH</strong>` over `<strong><span>0.1 ETH</span></strong>` to keep the DOM simple. The span element doesn't add any value in this case